kubernetes/structures_container.go

```diff
@@ -1016,6 +1016,195 @@ func expandConfigMapRef(r []interface{}) (*v1.ConfigMapEnvSource, error) {
 	return obj, nil
 }
 
+func expandEnv(e []interface{}) ([]map[string]interface{}, error) {
```

Comment:
Let's put these functions in their own file for now, call it `structures_env.go`